mlrun/datastore/sources.py

```diff
@@ -747,7 +747,7 @@ class SnowflakeSource(BaseSourceDriver):
             url="...",
             user="...",
             database="...",
-            schema="...",
+            db_schema="...",
```

Comment:
Need to change this in docs also.

Reply:
@jillnogold will change it in https://github.com/mlrun/mlrun/pull/5569 - snowflake docs pr:)

Reply:
It can easily be forgotten. It's just a tiny change in two places... Better just do it here.